provider/blob_online.go

```diff
@@ -69,16 +78,28 @@ func (store OnlineFileStore) readTableValue(feature, variant string) (ValueType,
 
 func (store OnlineFileStore) writeTableValue(feature, variant string, valueType ValueType) error {
 	tableKey := blobTableKey(store.Prefix, feature, variant)
-	return store.Write(tableKey, []byte(valueType.Scalar()))
+	filepath := filestore.AzureFilepath{}
+	if err := filepath.ParseFilePath(tableKey); err != nil {
+		return err
+	}
+	return store.Write(&filepath, []byte(valueType.Scalar()))
 }
 
 func (store OnlineFileStore) deleteTable(feature, variant string) error {
 	tableKey := blobTableKey(store.Prefix, feature, variant)
 	entityDirectory := entityDirectory(store.Prefix, feature, variant)
-	if err := store.Delete(tableKey); err != nil {
+	filepath := filestore.AzureFilepath{}
+	if err := filepath.ParseFilePath(tableKey); err != nil {
+		return err
+	}
+	if err := store.Delete(&filepath); err != nil {
 		return fmt.Errorf("could not delete table index key %s: %v", tableKey, err)
 	}
-	if err := store.DeleteAll(entityDirectory); err != nil {
+	filepath = filestore.AzureFilepath{}
```

Comment:
Typically I would have a filestore.ParseAzureFilePath top level function for each of these rather than instantiating in this way then calling Parse